docs/tutorials/text-classification-tutorial.mdx

```diff
@@ -0,0 +1,295 @@
+| 1 | 0 | 3 | 0 | !!!!! RT @mleew17: boy dats cold...tyga dwn bad for cuffin dat hoe in the 1st place!! |
+
+---
+
+## Creating a Predictor
+
+Now that you're connected to MySQL, it's time to create your predictor model.
+
+- First, navigate to the mindsdb database:
+
+ `use mindsdb;`
+
+Your output should read:
+`Query successfully completed`
+
+- Next, make sure your data is there with SQL's `SELECT` statement in your MindsDB Cloud Editor Instance:
+
+```sql
+SELECT *
+FROM twitter_data.tweets
+LIMIT 3;
+```
+***your output will look like this, but with 10 rows***
+
+| id  | hate_speech | offensive | neither | tweet |
+| --- | ----------- | --------- | ------- | ----- |
+| 0 | 0 | 0 | 3 | !!! RT @mayasolovely: As a woman you shouldn't complain about cleaning up your house. &amp; as a man you should always take the trash out... |
+| 1 | 0 | 3 | 0 | !!!!! RT @mleew17: boy dats cold...tyga dwn bad for cuffin dat hoe in the 1st place!! |
+
+---
+
+- Now, it's time to create your prediction model!
+
+```sql
+CREATE MODEL mindsdb.hf_test
+PREDICT PRED
+USING
+engine = 'huggingface',
+task = 'text-classification',
+model_name = "Hate-speech-CNERG/bert-base-uncased-hatexplain",
+input_column = 'tweet',
+labels = ['hate_speech', 'neither', 'offensive'];
+```
+
+| hf_test | huggingface | mindsdb | true | 1 | generating | [NULL] | PRED | up_to_date | 23.3.4.0 | [NULL] | [NULL] | {'target': 'PRED', 'using': {'task': 'text-classification', 'model_name': 'Hate-speech-CNERG/bert-base-uncased-hatexplain', 'input_column': 'tweet', 'labels': ['hate_speech', 'neither', 'offensive']}}  | [NULL] |
+
+
+## Checking The Status of a Predictor
+
+- Next up, check the status of your predictor:
+
+```sql
+SELECT *
+FROM mindsdb.models 
+WHERE name = 'hf_test';
+```
+
+| hf_test | huggingface | mindsdb | 1 | complete | [NULL] | PRED | up_to_date | 23.3.5.0 | [NULL] | [NULL] | {'target': 'PRED', 'using': {'task': 'text-classification', 'model_name': 'Hate-speech-CNERG/bert-base-uncased-hatexplain', 'input_column': 'tweet', 'labels': ['hate_speech', 'neither', 'offensive']}} | [NULL] | [NULL] | [NULL] | [NULL] | 2023-03-29 17:15:38.683663 |
+
+---
+
+`DESCRIBE PREDICTOR mindsdb.hf_test;`
+
+```sql
+{"input_column":"tweet","labels":["hate_speech","neither","offensive"],"labels_map":{"hate speech":"hate_speech","normal":"neither","offensive":"offensive"},"max_length":512,"model_name":"Hate-speech-CNERG/bert-base-uncased-hatexplain","target":"PRED","task":"text-classification","task_proper":"text-classification"}
+```
+
+## Training a Predictor
+
+Now that your model is finished generating, you can go ahead and test it out! ***Note: In order to confirm your new model's efficiency, you will most likely use some language you don't like or agree with here. Just keep in mind, you can ultimately use this highly effective technology to do good! Do not follow the tutorial if you intent to use this malicously.***
```

Comment:
Please remove all offensive/vulgar words and examples from this tutorial. MindsDB documentation shouldn't contain any negative language or hate speech.